agimus_controller/tests/test_mpc_data.py

```diff
@@ -62,7 +62,7 @@ def test_ocp_debug_data(self):
                 f"Value missmatch for parameter '{key}'. Expected: '{val}', got: '{res}'",
             )
 
-    def test_ocp_debug_data(self):
+    def test_mpc_debug_data(self):
```

Comment:
This does not seem related to ruff

Reply:
yes it is, ruff detected this error. So I fixed this error.

Reply:
Ok! Didn't know ruff could do that, my bad.